pkg/storage/bolt.go

```diff
@@ -141,6 +142,108 @@ func (b *BoltDB) DeleteNamespace(namespace string) error {
 	})
 }
 
+// UpdaterWithMap is a json map based Updater implementation. The key/values from the map are used to update the
+// unmarshalled JSON representation of the stored data.
+type UpdaterWithMap struct {
+	Values map[string]any
+}
+
+func (u UpdaterWithMap) Validate(v []byte) error {
+	return nil
```

Comment:
hmm, is there any validation that can be done here?

Reply:
sounds good to me. may be worth adding a comment to capture that for future readers wondering if this can be removed

Reply:
Added comment.